### ISSUE TITLE
Misleading or bad browser version comparision

### ISSUE THREAD

**Issue by bartTC** [lancedikson/bowser#217]
The readme states that I can compare a browser version with a greater/lower statement:

```
if (bowser.msie && bowser.version <= 6) {
  alert('Hello China');
}
```

But `bowser.version` as well as `bowser.osversion` is a string, this only works due to JS typecasting. It's not reliable and failed for my Google Pixel phone with version 8.0.2

```
>>> "8" > 6
true
>>> "8.0" > 6
true
>>> "8.0.2" > 6
false
```

**Comment by bartTC** [comment]
As a workaround I'm extracting the major version number manually and compare it.

```
let majorVersion = bowser.version.split('.')[0];
let majorOSVersion = bowser.osversion.split('.')[0];

if (majorVersion > 1 && majorOSVersion < 2) ...
```

**Comment by lancedikson** [comment]
Hi, @bartTC. Sorry for the delay. Yes, there is no a really cool way to compare versions still, but you can try this one:
```
bowser.compareVersions([bowser.version, '10']);
```
https://github.com/lancedikson/bowser#bowsercompareversionsversionsarraystringnumber

**Comment by lancedikson** [comment]
Fixed in 2.0.0-alpha.1
